### DISCUSSION TITLE
Copy all to left/right shortcut

### DISCUSSION THREAD

**Discussion by aianau** [WinMerge/winmerge#2115] [Ideas]
Is that a way that we can add a shortcut for the `Copy All to Right`/`Copy All to Left` buttons? 
I was thinking of the command Alt+A+LeftArrow/RightArrow.  (

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not have the capability to assign shortcut keys to arbitrary menus. However, if English is selected as the language, you can execute 'Copy All to Left' by pressing Alt+M → Y, and 'Copy All to Right' by pressing Alt+M → A.
